src/stories/Library/Lists/list-empty/list-empty.scss

```diff
@@ -6,7 +6,7 @@
   justify-content: center;
   align-items: center;
   border: 1px dotted $color__global-grey;
-  text-align: left;
+  text-align: center;
```

Comment:
Is this how it is defined in Figma?

Reply:
Yessir! :) @spaceo 
https://www.figma.com/file/Zx9GrkFA3l4ISvyZD2q0Qi/Designsystem?type=design&node-id=8513%3A86100&mode=dev&t=ZLRabCO2WoSJS8G9-1